OrbitLinuxTracing/LinuxTracingUtils.h

```diff
@@ -31,6 +31,8 @@ std::vector<pid_t> GetAllTids();
 
 std::string GetThreadName(pid_t tid);
 
```

Comment:
Since it returns char - add a comment or link explaining how to decode it.

Reply:
Good point, added links.